tox.ini

```diff
@@ -41,17 +41,15 @@ passenv =
   AZURE_PROD_STORAGE_ACCESS_KEY
   AZURE_DEV_STORAGE_ACCOUNT
   AZURE_DEV_STORAGE_ACCESS_KEY
-  AWS_TESTING_ACCESS_KEY_ID
-  AWS_TESTING_SECRET_ACCESS_KEY
-  AWS_PROD_ACCESS_KEY_ID
-  AWS_PROD_SECRET_ACCESS_KEY
-  AWS_DEV_ACCESS_KEY_ID
-  AWS_DEV_SECRET_ACCESS_KEY
+  AWS_ACCESS_KEY_ID
+  AWS_SECRET_ACCESS_KEY
+  AWS_SESSION_TOKEN
+  AWS_PROFILE
+  AWS_CONFIG_FILE
```

Comment:
It will be great, if we have a comment with references in this file. I think, `.ini` files allow it.
If someone starts the read from the `.ini` file, they are going to miss the understanding of the behavior and fallbacks that is used in the code.